extension/src/project/createProject.ts

```diff
@@ -19,17 +16,23 @@ export async function getNewProjectFilePath() {
         }
     };
 
-    let fileUri = await vscode.window.showSaveDialog(options);
-    if (fileUri)
-        return Promise.resolve(fileUri);
-
-    return Promise.resolve(undefined);
+    let fileUri = await vscode.window.showSaveDialog(options);    
+    if (fileUri) {
+        if (path.basename(fileUri.fsPath).indexOf(' ') === -1) {
+            return Promise.resolve(fileUri);
+        } else {
+            let msg = AutoLispExt.localize("autolispext.project.createproject.nospaces", "Legacy PRJ naming rules do not allow spaces");
```

Comment:
not sure the "`nospaces`" is in AutoLispExt/i18n/enu/out

Reply:
I perceived these arguments to localize to be Key & EnglishTranslation. Every time I create a new piece of user-facing text then it needed a new ID; Is that incorrect?
My assumption was the Autodesk Localization team would be using some kind of a tool like vscode-nl-dev to collect and create translations for these and all I really needed to do was make sure I use the localize() to give them that opportunity.

Reply:
They have no tools, and it only translate all the json style strings in the folder `AutoLispExt/i18n/enu`. So for developer we need to put a key-value json pair in that folder.

Reply:
That sounds like an extensive treasure hunt across the code base. IE, I believe we have many Keys in the code that do not existing json entries. I've certainly added to them by not understanding localize wasn't creating harvesting opportunities.
I will either be locating those tools or creating them if I necessary, but I am certain some kind of diffs tool is out there already.

Reply:
Some strings in the current repro are not handled by` localize()`, because developer think that this is impossible to have such case, so it doesn't do localize. I think there is only one case with `TODO ` which need to add to json file.

It is best if there is a tool to extract it and put it to json file.